README.md

```diff
@@ -1,2 +1,91 @@
-# Hello-world
-This is my official Microverse first repository
+<a name="readme-top"></a>
+<div align="center">
+
+  <h3><b>My first Microverse Repo</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+
```

Comment:
- [ ] Please make sure that you add the [Contributing](https://github.com/microverseinc/readme-template#contributing), [ Show your support](https://github.com/microverseinc/readme-template#support), [ Acknowledgements](https://github.com/microverseinc/readme-template#acknowledgements) and [ License](https://github.com/microverseinc/readme-template#license) sections to your README file. since it is mandatory to add them to your README .